# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gmp/avr32.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- gmp-4.2.2/config.guess.vanilla	2007-09-22 22:58:48.000000000 +0200
+++ gmp-4.2.2/config.guess	2007-09-22 23:01:00.000000000 +0200
@@ -207,6 +207,10 @@
   exact_cpu=mips64
   ;;
 
+avr32-*-*)
+  exact_cpu=avr32
+  ;;
+
 m68k-*-*)
   # NetBSD (and presumably other *BSD) "sysctl hw.model" gives for example
   #   hw.model = Apple Macintosh Quadra 610  (68040)
